package.cmake

```diff
@@ -24,10 +24,10 @@ macro(setSofaFilePath name value)
 endmacro()
 ######################
 
-setSofaString(CMAKE_BUILD_TYPE Release)
```

Comment:
What's the default build type without this?

Reply:
Default is Release. This was used to force Release when packaging but I prefer letting the user choose what type of package he wants (we never know).